### ISSUE TITLE
How to manage monitoring sites with missing data for more than 75% of the study period 

### ISSUE THREAD

**Issue by ivanhanigan** [swish-climate-impact-assessment/BiosmokeValidatedEvents#8]
@farhadsalimi we need to address this concern from Geoff at some stage.  

Geoff says: 
For eg there are 4 pm2.5 monitoring sites operating during our study period.  the study period is very long - 20 years. during this period pm2.5 monitoring has changed within the sydney region.  i'm not sure about excluding PM2.5 monitors if data is not available from a site for >25% (ie: >5 years) of study period when we have such a long study period   this means we can exclude sites with 15 years of data.  as monitoring PM2.5 became more important the nsw epa likely increased the number of monitoring sites over time.  as we have such a long study period i think we should create 2 (or 3 at most) exposure metrics based on different monitor site exclusion criteria
1. site operating for >75: ie our current approach
2. best use of available data

re 2: look at operating periods of monitoring sites and base exclusions on reasonable data use criteria.  ideally this would result in 2 (or 3 at most)  periods where the number of monitoring sites used was potentially different.  eg: 94 to 2007? when 3 monitoring sites were operating for >75% of the time.  2008 to 2013 when 4 monitoring sites were operating for >75% of the time.  ideal;ly these periods will coincide with the periods for which the fire smoke day validation process was different, and the 95%ile/ 99%ile calculation to id those days will be different

the bottom line is even if this will not change any results, these alternative metrics will be useful to have. 

SO:
We think it would be useful to produce two pm2.5 metrics:
- one based on avaraging 3 sites for who study period as you suggest
- one for a sensitivity analysis based on:  3 sites for the period 1jan94 to chullora start date; 4 sites for the period chullora start date to 31dec2013.

call the pm2.5 estimates something like
- pm2.5site3
- pm2.5site34
